doc/generated/functions.gen

```diff
@@ -1197,6 +1197,10 @@ Multiple calls to
 <function xmlns="http://www.scons.org/dbxsd/v1.0">Default</function>
 are legal,
 and add to the list of default targets.
+As noted above, both forms of this call affect the
```

Comment:
Maybe remove "As noted above" and leave "Both forms.."

Reply:
I can change that, but it's not an edit I made in this PR, it's just the result of a build of the generated files picking up something from an earlier change (yes, one of mine).

Reply:
ahh gotcha. I'll merge. Probably worth a full review of docs at some point for wording. (in my spare time.. ;)